packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -780,6 +784,13 @@ export default class TabAdvanced extends Component {
                   labelId='vmWarmMigration'
                   tooltip={isWarmMigrationAvailable ? undefined : _('availableXoaPremium')}
                 />
+                <TabButton
+                  btnStyle='warning'
+                  handler={isMigrationAllowed ? vmBlockMigration : vmAllowMigration}
```

Comment:
We don't want to prevent the user from unblocking the migration, even if it has been blocked by the backup. 

For instance, there might be an issue with the backup which prevented it from unblocking the operation at then end, and we want users to be able to bypass this.

But I think it would make sense to display the current reason it is blocked before unblocking, maybe by showing a modal (and only if the reason is not trivial, i.e. `'true'`).

Reply:
@julien-f Can we already ship the feature like this, and maybe improve it later? 
Or do you want it to be improved now?

Reply:
I fear that doing it in two steps will in fact make it more work and take longer.

IMHO that adding a quick confirm should be quick, what do you think @pdonias (of both the idea and how to implement it)?

Reply:
@pdonias Do you think that someone could help on the front-side?

Reply:
@b-Nollet There is no TabButton anymore, can you resolve the conversation please?